MAINTAINERS.md

```diff
@@ -3,12 +3,8 @@ Project Lead:
 
 Maintainers:
 
-@ashcrow
-@axelsimon
-@font
-@lukehinds
```

Comment:
It's possible that @lukehinds wants to still be involved in this, so I'll wait for him to signal his thoughts. But other than that, the changes lgtm.